api/src/localstack/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.8'
+
+# Note: did not work well on Apple M1
+services:
+  localstack:
+    container_name: '${LOCALSTACK_DOCKER_NAME-localstack_main1}'
+    image: localstack/localstack
+    network_mode: bridge
+    ports:
+      - '127.0.0.1:4566:4566'
+      - '127.0.0.1:4571:4571'
+    environment:
+      - SERVICES=serverless,cognito,rds,sqs,sns,cloudformation
```

Comment:
Do we need all services? This will put load on localstack.

Reply:
You're right. I couldn't quite test this as it anyway did not work on my M1. I will update only to keep 'serverless'.